src/main/webapp/app/assessment/unreferenced-feedback-detail/unreferenced-feedback-detail.component.ts

```diff
@@ -39,6 +42,20 @@ export class UnreferencedFeedbackDetailComponent {
 
     constructor(public structuredGradingCriterionService: StructuredGradingCriterionService) {}
 
+    ngOnInit() {
+        this.loadLongFeedback();
+    }
+
+    /**
+     * Call this method to load long feedback if needed
+     */
+    public async loadLongFeedback() {
+        if (this.feedback.hasLongFeedbackText) {
+            this.feedback.detailText = await this.feedbackService.getLongFeedbackText(this.resultId, this.feedback.id!);
+            this.onFeedbackChange.emit(this.feedback);
+        }
+    }
```

Comment:
Why are we not directly sending the longFeedbackText with the feedback? 
I think it would be a better approach to fetch the longFeedbackText directly from the server when fetching the feedback.

This has the potential to slightly increase the performance load on the server 😅

Reply:
its actually pretty rare that long feedback exists, i do not really see the point of sending a request everytime. we should only send it when a long feedback also exists